book/src/getting_started.md

```diff
@@ -25,17 +40,79 @@ Rust compiler. Here are the system requirements (they're pretty modest):
 
 [ru]: https://www.rustup.rs/
 
-There are two ways to get started working with Amethyst:
+## Rust project basics
+
+**If you consider yourself a rust veteran you can probably skip this section,
+just create a binary project and add a dependency for amethyst from crates.io.**
+
+Once you have that setup you'll need to make a new cargo binary project for your
+game to be written in.  You can do this with the following command
+```
```

Comment:
Some stricter markdown specs ask for an empty line before and after each code block, so I'd recommend doing that.

Reply:
Oh, thanks!